.github/workflows/schemathesis.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
         with:
           python-version: '3.8'
 
-      - run: pip install schemathesis
+      - run: pip install schemathesis httpx
```

Comment:
I think we do not need this anymore since schemathesis got fixed.
See: https://github.com/schemathesis/schemathesis/issues/1614

Reply:
Thanks for pointing that out!

fixed in [1ea4a10](https://github.com/bundesAPI/smard-api/pull/15/commits/1ea4a10443f60bb8fc5f80d9e20d1447b231b7ed)